content/the-incoming-summer-of-clippy.md

```diff
@@ -0,0 +1,61 @@
++++
+path = "9999/12/31/the-incoming-summer-of-clippy"
+title = "The incoming Summer of Clippy"
+authors = ["blyxyas"]
+
+[extra]
+team = "the Clippy team"
+team_url = "https://www.rust-lang.org/governance/teams/dev-tools#team-clippy"
++++
+
+We are currently working on a feature freeze for the Clippy project. This
+period would be 12 weeks long, starting on the introduction of 1.89 to beta
+on the 26 of June, and ending on the 18th of September.
```

Comment:
So, after reading the whole blog post I understand that from end of June until mid-September Clippy won't accept new feature requests but only fixes to existing lints - is that correct?

I think a TL;DR; of this kind could help, since the title is nice :sunglasses:  but doesn't explicit what is really being communicated.

How about this slightly reworded version:
```suggestion
The Clippy project will be on feature-freeze for 12 weeks, starting from Rust 1.89.0 beta (June 26th 2025) to September 18th 2025 (Rust 1.89.0 stable release). During this time no new features will be accepted, only bug fixes.
```

Reply:
I agree that this might be clearer to read, maybe not an explicit TL;DR because it's less than 500 words.